includes/wikia/traits/TitleTrait.php

```diff
@@ -152,4 +152,10 @@ public function getIndirectLinks( $level = 0 ) {
 
 		return $rows;
 	}
+
+	public function isTemplateDraft() {
+		global $wgEnableTemplateDraftExt;
+
+		return !empty( $wgEnableTemplateDraftExt ) && TemplateDraftHelper::isTitleDraft( $this );
```

Comment:
I think that the `isTitleDraft` logic should actually be placed in this function. You can leave `TamplateDraftHelper::isTitleDraft()` as a wrapper for it so we don't change every usage of it.